test/e2e/package.json

```diff
@@ -60,7 +60,7 @@
 		"spec-xunit-reporter": "./lib/reporter",
 		"testarmada-magellan": "11.0.1",
 		"testarmada-magellan-local-executor": "^2.0.0",
-		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#660cf2c",
+		"testarmada-magellan-mocha-plugin": "github:Automattic/magellan-mocha-plugin#fa17dd9",
```

Comment:
`fa17dd9` is related to specific commit in Automattic/magellan-mocha-plugin repo, right? Should this be removed when it goes to master branch or to stay like this?